### ISSUE TITLE
[Feature Request] Options to set objects' height and width manually

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [SVG-Edit/svgedit#836]
Although SVG-edit can resize an object by dragging it, the editor have no option to resize by inputting width and height, which is even implemented by Boxy SVG. With this feature user would be able to set objects’ size as explicited, and this option should also have an option to keep aspect ratio.

**Comment by jfhenon** [comment]
I'm not sure I get your point. If you select an object, you should be able to force the object size manually in the top toolbar.
For examole: 
<img width="956" alt="image" src="https://user-images.githubusercontent.com/20402845/193558772-87816d3e-8c89-4136-824d-9f367c66f18d.png">

**Comment by KrasnayaPloshchad** [comment]
Yes, I see, but there is still no option to keep aspect ratio, which could be a check box, or a lock icon between width and height inpot boxes.

**Comment by KrasnayaPloshchad** [comment]
And there is no way to set for group, path ot multi-selections.
